src/main/java/net/dv8tion/jda/internal/entities/GuildImpl.java

```diff
@@ -473,6 +475,47 @@ public TextChannel getRulesChannel()
         return rulesChannel;
     }
 
+    @Nonnull
+    @Override
+    public @NotNull RestAction<GuildScheduledEvent> retrieveScheduledEventById(String id)
+    {
+        Checks.isSnowflake(id);
+        return new DeferredRestAction<>(getJDA(), GuildScheduledEvent.class,
+                () -> getScheduledEventById(id),
+                () ->
+                {
+                    Route.CompiledRoute route = Route.Guilds.GET_SCHEDULED_EVENT.compile(getId(), id);
+                    return new RestActionImpl<>(getJDA(), route, (response, request) -> api.getEntityBuilder().createGuildScheduledEvent(this, response.getObject(), getIdLong()));
+                });
+    }
+
+    @Nonnull
+    @Override
+    public @NotNull RestAction<GuildScheduledEvent> retrieveScheduledEventById(long id)
```

Comment:
Use javax's NonNull

Reply:
these are javax's Nonnull, same as is used on all the other methods in the entire class from what I see

Reply:
the redundant @NotNull were removed